proto/src/determined/api/v1/api.proto

```diff
@@ -180,6 +180,17 @@ service Determined {
         option (google.api.http) = {post: "/api/v1/preview-hp-search" body: "*"};
         option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {tags: "Experiments"};
     }
+
+    // Get the list of trials for an experiment.
+    rpc GetExperimentTrials(GetExperimentTrialsRequest) returns (GetExperimentTrialsResponse) {
+        option (google.api.http) = {get: "/api/v1/experiments/{experiment_id}/trials" };
+        option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {tags: ["Trials", "Experiments"]};
+    }
+    // Get a single trial.
+    rpc GetTrial(GetTrialRequest) returns (GetTrialResponse) {
+        option (google.api.http) = {get: "/api/v1/trials/{trial_id}" };
+        option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {tags: ["Trials", "Experiments"]};
```

Comment:
Should this have the `"Experiments"` tag?

Reply:
it does, doesn't it? : )

Reply:
easy to update whenever